app/models/user.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+class User extends AppModel
+{
+    public function getUserDetail($id)
+    {
+        $db = DB::conn();
+        $row = $db->row('SELECT * FROM user WHERE id = ?', array($id));
+        return $row;
+    }
+    public function getUserName($user_id)
+    {
+        $db = DB::conn();
+        $row = $db->row('SELECT username FROM user WHERE id = ?', array($user_id));
+        return $row['username'];
```

Comment:
I would recommend to make use of `getUserDetail`
ex.
```php
$user = $this->getUserDetail($user_id);
return $user->name;
```
if user does not exist, an exception will be thrown then